tests/functional/aws-node-sdk/test/versioning/objectCopy.js

```diff
@@ -133,6 +134,45 @@ testing('Object Version Copy', () => {
             });
         }
 
+        it('should copy the appropriate version into the same bucket',
+        () => s3.putObjectAsync({ Bucket: sourceBucketName,
+              Key: sourceObjName,
+              Body: secondContent })
+            .then(() => s3.copyObjectAsync({
+                Bucket: sourceBucketName,
+                CopySource: `${sourceBucketName}/${sourceObjName}?` +
+                `versionId=${versionId}`,
+                Key: destObjName,
+            }))
+            .then(() => s3.getObjectAsync({ Bucket: sourceBucketName,
+              Key: destObjName }))
+            .then(res => assert.strictEqual(res.Body.toString(), content))
+            .catch(err => {
+                process.stdout.write('Error putting/copying object: ' +
+                `${err}\n`);
+                throw err;
+            }));
+
+        it('should copy the appropriate version into a different ' +
+        'destination bucket',
+        () => s3.putObjectAsync({ Bucket: sourceBucketName,
+              Key: sourceObjName,
+              Body: secondContent })
+            .then(() => s3.copyObjectAsync({
+                Bucket: destBucketName,
+                CopySource: `${sourceBucketName}/${sourceObjName}?` +
```

Comment:
same as above, it should be the same as the `copySource` variable set in the beforeEach, right?